src/consumer.ts

```diff
@@ -25,12 +27,14 @@ export class StreamConsumer implements Consumer {
       stream: string
       consumerId: number
       consumerRef?: string
+      offset: Offset
     }
   ) {
     this.client = params.client
     this.stream = params.stream
     this.consumerId = params.consumerId
     this.consumerRef = params.consumerRef
+    this.offset = params.offset
```

Comment:
I don't get why you added the offset property inside the Consumer, where is it used?

Reply:
we need the offset when sending the ConsumerUpdateResponse, so that we complete the promotion of single active consumers

Reply:
Right, my bad!